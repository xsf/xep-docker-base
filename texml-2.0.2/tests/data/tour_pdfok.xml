<TeXML>
<TeXML escape="0">
\documentclass{article}
\usepackage[T2A]{fontenc}
\usepackage[koi8-r]{inputenc}
\usepackage[unicode]{hyperref}
</TeXML>
<env name="document">
  <cmd name="section">
    <opt>
      <cmd name="texorpdfstring">
        <parm>Заголовок (Title)</parm>
        <parm><pdf>Заголовок (Title)</pdf></parm>
      </cmd>
    </opt>
    <parm>Заголовок (Title)</parm>
  </cmd>
  Текст (Text)
</env>
</TeXML>
