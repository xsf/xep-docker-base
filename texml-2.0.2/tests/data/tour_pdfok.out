\documentclass{article}
\usepackage[T2A]{fontenc}
\usepackage[koi8-r]{inputenc}
\usepackage[unicode]{hyperref}
\begin{document}
\section[ \texorpdfstring{Заголовок (Title)}{\004\027\004\060\004\063\004\076\004\073\004\076\004\062\004\076\004\072\000\040\000\050\000\124\000\151\000\164\000\154\000\145\000\051}
]{Заголовок (Title)} Текст (Text)
\end{document}
